pkg/kvm/kvm.go

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2023 Intel Corporation
+
+// Package kvm automates plugging of SPDK devices to a QEMU instance
+package kvm
+
+import (
+	"context"
+	"log"
+	"path/filepath"
+
+	"github.com/opiproject/opi-spdk-bridge/pkg/frontend"
+	"google.golang.org/protobuf/types/known/emptypb"
+
+	pb "github.com/opiproject/opi-api/storage/v1alpha1/gen/go"
+)
+
+// Server is a wrapper for default opi-spdk-bridge frontend which automates
+// interaction with QEMU instance to plug/unplug SPDK devices
+type Server struct {
+	*frontend.Server
+
+	qmpAddress string
+	ctrlrDir   string
+}
+
+// NewKvmServer creates instance of KvmServer
+func NewKvmServer(s *frontend.Server, qmpAddress string, ctrlrDir string) *Server {
```

Comment:
maybe just `NewKvmServer` ? we know it's KVM package

Reply:
correct. `NewServer` fits better